tests/kernel/fifo/fifo_timeout/prj.conf

```diff
@@ -1,3 +1,5 @@
 CONFIG_ZTEST=y
 CONFIG_IRQ_OFFLOAD=y
 CONFIG_MP_NUM_CPUS=1
+CONFIG_TIMESLICING=n
```

Comment:
@nashif what was the reason for this?

Reply:
trying to save some ram :)

Reply:
sure, i wonder though if this affects the behavior of the test, or limits its coverage. Otherwise, no complaints.

Reply:
no it does not, this is a fifo object test, not scheduler test. Slicing is tested somewhere else and everywhere, as it is enabled by default.